src/main/java/net/dries007/holoInventory/compat/InventoryDecoderRegistry.java

```diff
@@ -94,12 +94,21 @@ public NBTTagList toNBT(IInventory inv) {
                         return DEFAULT.toNBT(inv);
                     }
                     NBTTagList list = new NBTTagList();
-                    ItemStack stack = inv.getStackInSlot(1);
-                    if (stack != null) {
-                        NBTTagCompound tag = stack.writeToNBT(new NBTTagCompound());
-                        int item_amount = ((GT_MetaTileEntity_QuantumChest) metaTileEntity).mItemCount;
-                        tag.setInteger(NBT_KEY_COUNT, stack.stackSize + item_amount);
-                        list.appendTag(tag);
+                    for (int i = 0; i < 2; i++) {
+                        ItemStack stack = inv.getStackInSlot(i);
+                        if (stack != null) {
+                            NBTTagCompound tag = stack.writeToNBT(new NBTTagCompound());
+                            tag.setInteger(NBT_KEY_COUNT, stack.stackSize);
+                            list.appendTag(tag);
+                            if (i == 1) {
```

Comment:
`getStackInSlot(1)` is not necessarily the same as the item stored in the quantum chest
![2023-09-14_16 59 03](https://github.com/GTNewHorizons/HoloInventory/assets/40035906/d7b7a447-e700-464c-9eea-87115a81fece)
![2023-09-14_16 59 06](https://github.com/GTNewHorizons/HoloInventory/assets/40035906/06800e54-b66d-412f-972f-371aae0ca2cb)